src/Tools/dotnet-user-jwts/src/Helpers/DevJwtCliHelpers.cs

```diff
@@ -27,7 +27,7 @@ public static string GetOrSetUserSecretsId(string projectFilePath)
 
     public static bool GetProjectAndSecretsId(string projectPath, IReporter reporter, out string project, out string userSecretsId)
     {
-        var finder = new MsBuildProjectFinder(projectPath ?? Directory.GetCurrentDirectory());
+        var finder = new MsBuildProjectFinder(Directory.GetCurrentDirectory());
```

Comment:
Does the create command have the same issue?

Reply:
Yes, will update and add a test.